src/pages/UserHome.jsx

```diff
@@ -138,40 +138,42 @@ export default function UserHome() {
           </Button>
         </Grid>
       </Grid>
-      {error?.toString() ||
-      (loading && <CircularProgress className={classes.progress} />) ||
-      scholarships.length === 0 ? (
-        <Grid
-          container
-          component={Paper}
-          variant="outlined"
-          className={classes.noneAddedGrid}>
-          <Grid item>
-            <InboxIcon className={classes.inboxIcon} />
-          </Grid>
-          <Grid item>
-            <Typography variant="h5" gutterButtom>
-              No Scholarships Added Yet
-            </Typography>
-            <MuiLink component={Link} to="/scholarships/new">
-              Add Scholarship
-            </MuiLink>
-          </Grid>
-        </Grid>
+      {error?.toString() || loading ? (
+        <CircularProgress className={classes.progress} />
       ) : (
-        <>
-          <ScholarshipList scholarships={scholarships} />
-          {canLoadMore ? (
-            <Button
-              className={classes.loadMoreButton}
-              color="primary"
-              onClick={() => loadMoreScholarships(loadMoreFn)}>
-              Load More
-            </Button>
+        [
```

Comment:
What is the array for? I think it works without it?

Reply:
I don't think its an array, I think it was a way to do the nested conditional if/else thing I read it on stack overflow it gave warning

Reply:
Ok I'm fine merging it as is